twistr/space_race.go

```diff
@@ -0,0 +1,64 @@
+package twistr
+
+import "fmt"
+
+type SpaceId int
+
+const (
+	NoAbility SpaceId = iota
+	TwoSpace
+	OppHeadlineFirst
+	DiscardHeld
+	ExtraAR
+)
+
+type SRBox struct {
+	MaxRoll    int
+	OpsNeeded  int
+	FirstVP    int
+	SecondVP   int
+	SideEffect SpaceId
+}
+
+var SRTrack []SRBox = []SRBox{
+	SRBox{0, 0, 0, 0, NoAbility},
+	SRBox{3, 2, 2, 1, NoAbility},
+	SRBox{4, 2, 0, 0, TwoSpace},
+	SRBox{3, 2, 2, 0, NoAbility},
+	SRBox{4, 2, 0, 0, OppHeadlineFirst},
+	SRBox{3, 3, 3, 1, NoAbility},
+	SRBox{4, 3, 0, 0, DiscardHeld},
+	SRBox{3, 3, 4, 2, NoAbility},
+	SRBox{2, 4, 2, 0, ExtraAR},
+}
+
+func nextSRBox(s *State, player Aff) (srb SRBox, err error) {
+	playerPos := s.SpaceRace[player]
```

Comment:
just getting way into the nits here and not holding back or anything, but, do we really need to type out "playerPos"?

I can figure that shit out if you just call it, like, "pos"